internal/uploadmgr/gc.go

```diff
@@ -39,7 +39,7 @@ func (gc *bundleGC) Start(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	bundleInformer.AddEventHandler(toolscache.ResourceEventHandlerFuncs{
+	_, _ = bundleInformer.AddEventHandler(toolscache.ResourceEventHandlerFuncs{
```

Comment:
Hmm. I probably totally overlooked these return values originally. Anything we can/should do with either of these?

If not, it probably warrants a comment.

Reply:
Honestly, I was just trying to fix the lint without changing the behavior. But I could check the error.